pkgs/by-name/fi/firezone-gateway/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+  git,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "firezone-gateway";
+  version = "1.4.3";
+  src = fetchFromGitHub {
+    owner = "firezone";
+    repo = "firezone";
+    tag = "gateway-${version}";
+    hash = "sha256-BXfEQ5KQMZDiHgMr9+rfI650qeDeF1TgKIqFShz2Ztc=";
+    # FIXME: Remove once https://github.com/firezone/firezone/pull/8095 is in
+    postFetch = ''
+      ${lib.getExe git} -C $out apply ${./0000-load-system-ca-certs.patch}
+    '';
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-5L8jhT4qoF6gkSW0TD+fe114Kl/BZvVaFA5UoqQWudc=";
+  sourceRoot = "${src.name}/rust";
+  buildAndTestSubdir = "gateway";
+
+  # Required to remove profiling arguments which conflict with this builder
```

Comment:
Why? Building with framepointers is useful for many things not just profiling, including stack traces when things go wrong. Why can't nix not build with framepointers?

Reply:
I honestly don't know why it conflicts, but the `buildRustPackage` doesn't like it. Since this is a release build we won't have the full debug information in the binary anyway, so not adding framepointers doesn't make a huge difference. I'll add them back when I figure out why it conflicts.

Reply:
> Since this is a release build we won't have the full debug information in the binary anyway

But together with spit-debug info and frame-pointers, you still get nice stack traces.